utilities/openshift_resources/cluster_scoped_resource.py

```
@@ -0,0 +1,300 @@
+        """Return the local resource definition as a dict (without fetching from cluster)."""
+        return self._build_dict()
+
+    def create(self, from_dict: dict[str, Any] | None = None) -> Self:
+        """Create the resource. Returns self."""
+        return self._write(resource_dict=from_dict or self._kind_dict or self._build_dict(), verb="create")
+
+    def apply(self, resource_dict: dict[str, Any] | None = None) -> Self:
+        """Apply (create or update) the resource. Returns self."""
+        return self._write(resource_dict=resource_dict or self._build_dict(), verb="apply")
+
+    def output(self, format: str = "yaml") -> str:
```

Ruff (0.16.0)

[error] Function argument `format` is shadowing a Python builtin

(A002)